api/Navigator.json

```diff
@@ -2006,7 +2006,7 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-online-dev",
           "support": {
             "chrome": {
-              "version_added": "1",
+              "version_added": "2",
               "notes": "Earlier versions of Chrome incorrectly return true when a tab is first opened, but it starts reporting the correct connectivity status after the first network event. Windows: 11, macOS: 14, Chrome OS: 13, Linux: Always returns <code>true</code>. For history, see <a href='https://crbug.com/7469'>crbug.com/7469</a>."
```

Comment:
This note doesn't really make sense before or after this change, but oh well. We'd have to fix this if we do https://github.com/mdn/browser-compat-data/issues/17857.